teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] [Optional] It would be nice to have a getter method for the `specialization` attribute. This will make it possible for you to access the `specialization` attribute. It will be something like this:

`attr_reader :specialization`.